cmd/cluster-provisioner/main.go

```diff
@@ -32,37 +37,62 @@ func main() {
 	flag.Parse()
 
 	if *action == "provision" {
-		if err := provisioner.CreateVPC(context.TODO(), nil); err != nil {
+		fmt.Println("Creating VPC...")
```

Comment:
I wonder if the `Println` here would be better as log-rs logs instead? I appreciate that it's fitting in with some of the existing logging elsewhere, but maybe we should adapt that? @wainersm - do you have any thoughts on this?

Reply:
I agree @stevenhorsman I'd like to use a new PR to use logrus in the e2e test main package and the cluster-provisioner main package. Maybe even other CAA packages via yet another PR.